app.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+var hours = ['6am:', '7am:', '8am:', '9am:', '10am:', '11am:', '12pm;', '1pm:', '2pm:', '3pm:', '4pm:',  '5pm:', '6pm:', '7pm:', '8pm:'];
+
+function render(){
+  this.counts15Times();
+  this.heading = document.createElement('h2');
+  this.heading.textContent = this.location;
+  this.heading.setAttribute('id', this.id);
+  document.body.appendChild(this.heading);
+
+  this.img = document.createElement('img');
+  this.img.setAttribute('id', this.id);
+  this.img.setAttribute('src', this.imgSrc);
+  this.img.setAttribute('alt', this.location);
+  document.body.appendChild(this.img);
+
+  this.list = document.createElement('ul');
+  this.list.setAttribute('id', this.id);
+
+  for (var i = 0; i < this.avgCookiesADay.length; i++){
+    var cookiesanhourLi = document.createElement('li');
+    cookiesanhourLi.setAttribute('class', 'avgCookiesADay');
+    this.list.appendChild(cookiesanhourLi);
+  }
+}
+
+var firstAndPike = {
+  id: '1stnPike',
+  img:null,
+  list: null,
+  header: null,
+  location: '1st and Pike',
+  imgSrc:'https://media-cdn.tripadvisor.com/media/photo-s/09/d2/bd/6d/original-starbucks.jpg',
```

Comment:
the images are cool, but you're currently just attaching images and it's to both index.html and sales.html.
We only need the table on sales.html and if you want pictures, they can go on index.html. It's really neat that you can do the images like this, but you aren't separating them out for different pages.